packages/easy-testing/src/DataProvider/StaticFixtureFinder.php

```diff
@@ -18,17 +19,51 @@ final class StaticFixtureFinder
 {
     public static function yieldDirectory(string $directory, string $suffix = '*.php.inc'): Iterator
     {
-        $fileInfos = self::findFilesInDirectory($directory, $suffix);
+        return self::yieldFileInfos(self::findFilesInDirectory($directory, $suffix));
```

Comment:
Could you extract the fileinfos to own variable? Nested static call starts to be harder to read